koku/masu/processor/ocp/ocp_report_charge_updater.py

```diff
@@ -179,18 +176,37 @@ def _write_to_temp_table(self, report_accessor, charge_data):
         report_accessor.bulk_insert_rows(csv_file, temp_table, ['lineid', 'charge'])
         return temp_table
 
-    def _update_markup_cost(self):
-        """Store markup costs."""
-        try:
-            with CostModelDBAccessor(self._schema, self._provider_uuid,
+    def _update_markup_cost(self, start_date, end_date):
+        """Populate markup costs for OpenShift.
+
+        Args:
+            start_date (str) The date to start populating the table.
+            end_date   (str) The date to end on.
+
+        Returns
+            None
+
+        """
+        aws_markup_value = 0.0
+        infra_map = self.get_infra_map()
+        infra_tuple = infra_map.get(self._provider_uuid)
+        cluster_id = get_cluster_id_from_provider(self._provider_uuid)
+        if infra_tuple:
+            aws_uuid = infra_tuple[0]
+            with CostModelDBAccessor(self._schema, aws_uuid,
                                      self._column_map) as cost_model_accessor:
                 markup = cost_model_accessor.get_markup()
-                markup_value = float(markup.get('value', 0)) / 100
+                aws_markup_value = float(markup.get('value', 0)) / 100
+        with CostModelDBAccessor(self._schema, self._provider_uuid,
+                                 self._column_map) as cost_model_accessor:
+            markup = cost_model_accessor.get_markup()
+            ocp_markup_value = float(markup.get('value', 0)) / 100
 
-            with OCPReportDBAccessor(self._schema, self._column_map) as report_accessor:
-                report_accessor.populate_markup_cost(markup_value, self._cluster_id)
-        except OCPReportChargeUpdaterError as error:
-            LOG.error('Unable to update markup costs. Error: %s', str(error))
+        with OCPReportDBAccessor(self._schema, self._column_map) as accessor:
+            LOG.info('Updating OpenShift markup for'
+                     '\n\tSchema: %s \n\tProvider: %s \n\tDates: %s - %s',
+                     self._schema, self._provider_uuid, start_date, end_date)
+            accessor.populate_markup_cost(aws_markup_value, ocp_markup_value, cluster_id)
 
```

Comment:
Currently there is no indication that markup calculations complete.
```suggest
        LOG.info('Finished updating markup.')
```

Reply:
I suppose it could make more sense to put this log statement in the accessor instead.